packages/PyTrilinos/src/Domi.i

```diff
@@ -432,26 +433,42 @@ import numpy
 }
 %include "Domi_MDMap.hpp"
 #ifdef HAVE_TPETRA
-%template(getTpetraMap) Domi::MDMap::getTpetraMap< PYTRILINOS_LOCAL_ORD,
-                                                   PYTRILINOS_GLOBAL_ORD,
-                                                   DefaultNodeType >;
-%template(getTpetraAxisMap) Domi::MDMap::getTpetraAxisMap< PYTRILINOS_LOCAL_ORD,
-                                                           PYTRILINOS_GLOBAL_ORD,
-                                                           DefaultNodeType >;
-#endif
-%teuchos_rcp(Domi::MDMap< Domi::DefaultNode::DefaultNodeType >)
-%template(MDMap_default) Domi::MDMap< Domi::DefaultNode::DefaultNodeType >;
-%pythoncode
+%extend Domi::MDMap
 {
-MDMap = MDMap_default
+  // This extension returns a Tpetra::Classes::Map instead of a
+  // Tpetra::Map, to help SWIG with type checking
```

Comment:
I'm planning on purging `Tpetra::Classes::${CLASS}` soon, but this should be fine for now.  Does PyTrilinos actually get tested somewhere on the Dashboard?

Reply:
Not yet. My plan is to add my desktop machine to nightly testing for PyTrilinos. Each time I find time to work on it, I make incremental progress, but it is not there yet. Same for Domi.

Reply:
@wfspotz got it; thanks!